### PR TITLE
refactor(let paragraph in career inherit font)

### PR DESCRIPTION
**Pull request by milljoniaer**
### Why?

I recognized, that the Text of the Section Header on the Career Page was to small in Desktop Mode compared to Figma and the other Headers,

### What is included?

The local Paragraph Component inherits the font from its parent the Section Header, in that way it is responsive and has in each breakpoint the correct size.

### CODE REVIEW

File: src/page-building/career/career-page.tsx [hunk 1]
```diff
@@ -14,7 +14,6 @@ import { useTranslation } from 'gatsby-plugin-react-i18next';
 import { SyPersonioJob } from '../../types';
 
 const Paragraph = styled.p`
-  ${TextStyles.textR}
```
Comment (fischeversenker) [inline src/page-building/career/career-page.tsx:17]: Thanks for removing this. The following styles (space between adjacent paragraphs) is not needed as well. The template for this component only contains a single `<Paragraph>` so there is no point in keeping these styles. I suggest to completely remove this `Paragraph` and instead use a simple `<p>` in the template.

Comment (milljoniaer) [reply src/page-building/career/career-page.tsx:17]: I am not sure, but I think the Plan was to add multiple paragraphs.
For now there is only one, so I left it out. If we add more, we have to change it again, but I think it is less enough to remove it for now.

**Comment by milljoniaer** [comment]
I changed it just like Felix suggested it, please have another look at the PR.

File: src/page-building/career/career-page.tsx (hunk 1 above)
Comment (georgiee) [reply src/page-building/career/career-page.tsx:17]: I have removed those paragraphs in some PR before. While this change is okay I'm worrying about text paragraphs in general. We are not capable of freely writing texts right now 😕 but I think we can talk about that in https://github.com/satellytes/satellytes.com/issues/251

**Review by georgiee — APPROVED** [review]
let's merge, thanks

**Review by fischeversenker — APPROVED** [review]
Thanks for the discussion and the changes 🙌

**Pull request merged**